### PR TITLE
doc(tutorials): static websites (with videos)

### PR DESCRIPTION
**Pull request by bgever**
Adds a tutorial page for static websites, with two examples: 1) static HTML website, and 2) static Angular website. Both examples have an associated video (no audio), that quickly goes through the same process as explained in the text below.

### CODE REVIEW

File: docs/examples/static-website/tutorial.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+
+## Static HTML website
+
+<video autoplay loop muted controls width="100%">
+   <source src="/docs/assets/examples/static-websites/html.mp4"
+           type="video/mp4">
+   Sorry, your browser doesn't support embedded videos.
+</video>
+
+### Create a Website KintoBlock
```
Comment (Wabasi) [inline docs/examples/static-website/tutorial.md]: Is it intentional to have two "Create a website kintoblock" sections? Or is that extra.

**Comment by bappr** [comment]
@Disturbing @Wabasi what should we do with this one?
Bart is not here anymore.

**Comment by Disturbing** [comment]
@bappr need to fix the conflicts and merge IMO>

**Comment by bappr** [comment]
@ajesh-cc can you check that plz?

**Comment by ajesh-cc** [comment]
> @ajesh-cc can you check that plz?

@bappr : I have resolved the conflict.

File: docs/examples/static-website/tutorial.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+
+## Static Angular website
+
+<video autoplay loop muted controls width="100%">
+   <source src="/docs/assets/examples/static-websites/angular.mp4"
+           type="video/mp4">
+   Sorry, your browser doesn't support embedded videos.
+</video>
+
+### Create a Website KintoBlock
```
Comment (bappr) [inline docs/examples/static-website/tutorial.md]: @ajesh-cc there is 2 Create Website KintoBlock in that page. why?

Comment (ajesh-cc) [reply docs/examples/static-website/tutorial.md]: @bappr : Hi Ben. The second Create Website was added by Bart in the Quickstart Tutorial section. In my opinion the Quickstart content to create website is going to be helpful for beginners or starters.

Comment (bappr) [reply docs/examples/static-website/tutorial.md]: okay

**Pull request merged**